ants/lib/LOCAL_sliceImage.cxx

```diff
@@ -33,7 +33,7 @@ py::capsule sliceImage( py::capsule antsImage, int plane, int slice)
 
     filter->SetExtractionRegion( desiredRegion );
     filter->SetInput( itkImage );
-    filter->SetDirectionCollapseToSubmatrix();
+    filter->SetDirectionCollapseToGuess();
```

Comment:
what does this do?   just concerned about backward compatibility issues as I have lots of data processed the old way ( sub matrix )

Reply:
perhaps better to add a command line option and keep the old behavior as default